docs/community/governance.md

```diff
@@ -73,28 +73,77 @@ long experience with both the project and the larger ecosystem. When the core
 then admission to the SC by consensus. During that time deadlocked votes of the SC will
-be postponed until the new member has joined and another vote can be held.
-
-The napari steering council may be contacted at `napari-steering-council@googlegroups.com`.
-Or via the [@napari/steering-council](https://github.com/orgs/napari/teams/steering-council) GitHub team.
+be postponed until the new member has joined and another vote can be held. The IFP seat
+is elected by the IFP Advisory Council.
+
+The SC may be contacted at `napari-steering-council@googlegroups.com`.
+Or via the [@napari/steering-council](https://github.com/orgs/napari/teams/steering-council) GitHub team. For a list of the current SC see our [About Us](team) page.
+
+### Institutional and funding partners
+
+The SC is the primary leadership body for napari. No outside institution,
+individual or legal entity has the ability to own or control the project
+other than by participating in napari as contributors, core developers, and
+SC members. However, because institutions can be an important source of
+funding and contributions for the project, it is important to formally
+acknowledge institutional participation in the project. We call institutions
+recognized in this way Institutional and Funding Partners (IFPs).
+
+Institutions become eligible to become an IFP by employing individuals
+who actively contribute to napari as part of their official
+duties, or by committing significant funding to napari, as determined by the
+SC. Once an institution becomes eligible to become an IFP, the SC must
+nominate and approve the institution as an IFP. At that time one individual
+from the IFP is expected to become the IFP Representative and serve on the
+IFP Advisory Council. The role of the IFP Advisory Council is to provide
+input on project directions and plans, and to elect one IFP Representative
+to hold the IFP seat on the SC.
+
+The IFP Advisory Council is expected to self-organize according to rules
+agreed to by the existing IFPs. This document does not prescribe how IFPs
+should elect their representative on the SC, though the IFP Advisory Council
+should describe this process openly. IFPs are expected to work together
+and with the napari community in good faith towards a common goal
+of improving napari for the broader scientific computing community.
+
+If at some point an existing IFP is no longer contributing any employees
+or funding, then a one-year grace period commences. If during this one-year
+period they do not contribute any employees or funding, then at the end of
+the period their status as an IFP will lapse, and resuming it will require
+going through the normal process for new IFPs. If the IFP Representative on
+the SC is from an organization that loses its status as an Institutional Partner,
+that person will cease being a member of the SC and the remaining IFP Advisory
+Council members may choose a new Representative at their earliest convenience.
```

Comment:
should a vote arise while the IFPAC is in the process of electing a new SC representative, is this vote postponed until that member is admitted? In other words, can an SC vote go through without a currently sitting IFPAC committee member

Reply:
I would expect the SC and the IFPAC to work together in good faith to resolve this in a timely fashion